packages/components/Atoms/Icon/TextIcon.vue

```diff
@@ -15,6 +15,10 @@
 <script>
 import RplIcon from './Icon.vue'
 
+/**
+ * Text Icon allows user to display text with an icon that can be placed before or after.
+ * This is commonly used in an anchor <a> tag where the icon helps represents the action of the text.
+ */
```

Comment:
I feel like component description or component in general should be owned by UX/UI? Any suggestions to description?

Reply:
@rubyllamadora Yes, I will copy them from Ripple design file, you can find them in Zeplin.

Reply:
Thanks @tim-yao. Will look for them in zeplin and will update once I find the description.